src/components/entity/ha-chart-base.html

```diff
@@ -59,6 +60,10 @@
         overflow: hidden;
         box-sizing: border-box;
       }
+      .chartLegend li:first-of-type:last-of-type {
```

Comment:
What?! this is awesome. I did not know it existed. Cool!

Reply:
I wonder if we should just use flex-box to make the legend be drawn correctly? That seems more fool proof.

Reply:
I think multi-line flex would have the same effect. If we don't constrain the length of a single item we will often be get one item per line.
Flex can help with the last item in odd-long list, but it can be done with selectors too.